cmd/dial/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.redhat.io/rhel8/go-toolset as builder
+
+COPY main.go /opt/app-root/src/
+
+RUN CGO_ENABLED=0 go build -o dial -ldflags "-s -w" main.go
+
+FROM scratch
+
+COPY --from=builder --chown=1001:1001 /opt/app-root/src/dial /dial
```

Comment:
Don't you want to use ubi-mini/micro in here?

Reply:
@zroubalik `scratch` should be even better I guess, there is less attack surface. Also I asked guy from productization and he said it's OK.

Reply:
Also for what we need I consider even micro obscenely big (not to mention minimal).

I could even achieve 100kB image with pure C instead of Go but `musl C lib` is not parrt of RHEL :disappointed: 

I've chosen Go since productization is easy with it.

Reply:
without `musl` I cannot create static binary with C

Reply:
ACK, I thought that `scratch` will be a productization problem. So it is fine and please disregard this :)